lib/dal/src/migrations/U3410__workspace_integration.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE workspace_integrations
+(
+    pk                          ident primary key default ident_create_v1(),
+    slack_webhook_url           text NULL,
```

Comment:
I would add a name and then make this webhook_url

Reply:
gotcha!